src/main/java/spoon/reflect/visitor/printer/ElementPrinterHelper.java

```diff
@@ -271,18 +271,22 @@ public void writeHeader(List<CtType<?>> types, Collection<CtReference> imports)
 				} else if (ref instanceof CtExecutableReference) {
 					CtExecutableReference execRef = (CtExecutableReference) ref;
 					if (execRef.getDeclaringType() != null) {
-						printer.write("import static " + execRef.getDeclaringType().getQualifiedName() + "." + execRef.getSimpleName() + ";").writeln().writeTabs();
+						printer.write("import static " + this.removeInnerTypeSeparator(execRef.getDeclaringType().getQualifiedName()) + "." + execRef.getSimpleName() + ";").writeln().writeTabs();
 					}
 				} else if (ref instanceof CtFieldReference) {
 					CtFieldReference fieldRef = (CtFieldReference) ref;
-					printer.write("import static " + fieldRef.getDeclaringType().getQualifiedName() + "." + fieldRef.getSimpleName() + ";").writeln().writeTabs();
+					printer.write("import static " + this.removeInnerTypeSeparator(fieldRef.getDeclaringType().getQualifiedName()) + "." + fieldRef.getSimpleName() + ";").writeln().writeTabs();
 				}
 
 			}
 			printer.writeln().writeTabs();
 		}
 	}
 
+	private String removeInnerTypeSeparator(String FQN) {
```

Comment:
Name `FQN` does not comply with Java conventions. Our xxxx-checkstyle doesn't detect this?